### ISSUE TITLE
some questions about the Rand error computation

### ISSUE THREAD

**Issue by thouis** [cremi/cremi_python#2]
I'm having trouble squaring the comments in the code and the code itself.

In this line, it says that background (zero) pixels are excluded from the computation:
https://github.com/cremi/cremi_python/blob/master/cremi/evaluation/rand.py#L13

In this line, the total number of pixels (zero and nonzero) are computed as "n":
https://github.com/cremi/cremi_python/blob/master/cremi/evaluation/rand.py#L42

And "n" is used here in the computation:
https://github.com/cremi/cremi_python/blob/master/cremi/evaluation/rand.py#L60-L61

I would expect n to be something like "np.count_nonzero(segA)"

**Comment by funkey** [comment]
You are right, `n` should only reflect the number of valid locations. However, none of the reference implementations of the [Adapted Rand Error](http://brainiac2.mit.edu/SNEMI3D/evaluation) (see also [here](https://github.com/janelia-flyem/gala/blob/master/gala/evaluate.py)) is doing so.

**Comment by thouis** [comment]
I think this can just be closed.  If the bug in #3 is fixed, then it doesn't matter how `n` is computed, as it falls out of the equation.
